### PR TITLE
python310Packages.mne-python: 1.0.3 -> 1.1.0

### PR DESCRIPTION
**Pull request by onny**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/mne-python/default.nix [hunk 1]
```diff
@@ -1,53 +1,62 @@
...
+, jinja2
+, pooch
+, tqdm
 }:
 
 buildPythonPackage rec {
   pname = "mne-python";
-  version = "1.0.3";
-
-  disabled = isPy27;
```
Comment (fabaff) [inline pkgs/development/python-modules/mne-python/default.nix:21]: This should be updated as upstream is limiting the their support to [>=3.7](https://github.com/mne-tools/mne-python/blob/main/setup.py#L101).

**Comment by onny** [comment]
Thank you for reviewing @fabaff :)

**Review by bcdarwin — APPROVED** [review]
Looks fine to me, thanks. [edit: see below]

**Review by bcdarwin — CHANGES_REQUESTED** [review]
The package now depends on setuptools at runtime, and so `import mne` fails (this is #133032).

**Comment by bcdarwin** [comment]
As a minor point, matplotlib is listed twice in the inputs.

**Comment by onny** [comment]
@bcdarwin Thanks for reviewing. Fixed the two remaining issues :)

**Review by bcdarwin — APPROVED** [review]

**Pull request merged**